### ISSUE TITLE
Add storybook and a11y testing

### ISSUE THREAD

**Issue by artaommahe** [artaommahe/akker#112]
Currently the project is small and simple enough to fell comfortable developing it without a storybook. But also this means that any a11y issues should be checked manually. It's time to add a storybook and enable automatic a11y tests

- add storybook with a11y addon
- add stories for all (most) components
- enable automatic a11y testing

related
- https://storybook.js.org/docs/writing-tests/accessibility-testing
- https://analogjs.org/docs/integrations/storybook
- https://storybook.js.org/docs/writing-tests/integrations/vitest-addon
- https://storybook.js.org/docs/writing-tests/in-ci
